src/app.json.template

```diff
@@ -10,12 +10,8 @@
         "odt"
     ],
     "companyName": "[ COMPANY NAME HERE ]",
-    "defaultLocale": "en-US",
-    "supportedLocales": [
-        "en-US",
-        "fr-FR"
-    ],
-    "minUploadSize": 30720,
+    "defaultLocale": "en-EU",
```

Comment:
Dropped the `supportedLocales`, to define new locales is documented in the README.md.